releasenotes/releasenotes_v5.rst

```diff
@@ -3,13 +3,24 @@
 Current Firely Server release notes (v5.x)
 ==========================================
 
-.. _vonk_releasenotes_5_0_0:
+.. _vonk_releasenotes_5_0_0-beta1:
 
 Release 5.0.0-beta1, January 19th, 2023
 ------------------------------
 .. attention::
     This is a beta release of Firely Server 5.0.0. Although the core functionality remains fully intact, parts of the public API have been removed or heavily modified. Please consult the list under section 'Plugin and Facade' and check whether your implementation is affected by these changes.
 
+Database
+^^^^^^^^
+
+#. Because of feature 6 below, the database was updated for both **SQL Server** and **MongoDB**. Firely Server will usually perform the upgrade automatically. See for details :ref:`migrations`.
```

Comment:
I would suggest to spell out what the feature is instead of saying „feature 6“

Reply:
done